pages/standards-guidelines/wcag/wcag3-intro.md

```diff
@@ -38,7 +38,7 @@ footer: >
 {% include box.html type="start" h="2" title="Summary" class="full" %}
 {:/}
 
-Many aspects of W3C Accessibility Guidelines (WCAG) 3 are in an exploratory phase, and will change substantially. We published an updated [WCAG 3 Working Draft](https://www.w3.org/TR/wcag-3.0/) on 12 December 2024.
+Many aspects of W3C Accessibility Guidelines (WCAG) 3 are in an exploratory or developng phase and will change substantially.
 
 WCAG 3 will have similar fundamental and specific accessibility requirements as WCAG 2. WCAG 3 will have a different structure, different conformance model, and broader scope.
```

Comment:
Rereading this, I would now change it to simply say "WCAG 3 will have similar accessibility requirements as WCAG 2."  We are not using "fundamental" any longer and I don't think "specific" adds anything.